types/wlr_output.c

```diff
@@ -206,7 +206,7 @@ bool wlr_output_set_cursor(struct wlr_output *output,
 		output->cursor.texture = wlr_render_surface_init(output->cursor.renderer);
 	}
 
-	wlr_surface_attach_pixels(output->cursor.texture, WL_SHM_FORMAT_XBGR8888,
+	wlr_surface_attach_pixels(output->cursor.texture, WL_SHM_FORMAT_ABGR8888,
```

Comment:
As mentioned in the first pr comment i stumbled across the same bug that was fixed by @acrisci a few days ago. I used abgr which i guess makes more sense than xbgr (since we are attaching a cursor which will usuallly have an alpha channel)? i know this does not belong here but could not really test the meson build without this when i wrote it.